Assets/Scripts/ColorSwap.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ColorSwap : MonoBehaviour
+{
+
+    [SerializeField] ColorRenderer target;
+
+    [SerializeField] Color[] colors;
+    // Start is called before the first frame update
+
+    public void swapColor(int i)
+    {
+        if (!target)
+            return;
+        target.color = colors[i];
```

Comment:
Change this into 

foreach( SpriteRenderer t in target)
{ t.color = colors[i]; }

That'll loop through each renderer added to the array of them. The medium thunder clouds use 2 Sprite renders, so this will make setting them up easier